verifyincluded.go

```diff
@@ -144,6 +140,17 @@ Note: for publicly attested events, or shared protected events, you must use --t
 				// get the massif index for the event event
 				massifIndex := massifs.MassifIndexFromMMRIndex(cmd.massifHeight, event.MMRIndex())
 
+				// find the log tenant path if not provided
+				if tenantLogPath == "" {
+
+					logTenant, err := event.LogTenant()
```

Comment:
why not just

     tenantLpgPath, err := event.LogTenant()

and ave using one extra variable?

Reply:
good shout!

Reply:
done!